.github/workflows/master-e2e.yaml

```diff
@@ -316,23 +311,19 @@ jobs:
           UPGRADE_TYPE: manual
           VM_INDEX: 2
         run: cd tests && make e2e-upgrade-node
-      - name: Upgrade node 3 (with managedOSVersionName method) to specified Teal version
-        if: inputs.test_type == 'cli'
-        env:
-          IMAGE_VERSION: teal-5.3
-          UPGRADE_TYPE: managedOSVersionName
-          VM_INDEX: 3
-        run: cd tests && make e2e-upgrade-node
+#     - name: Upgrade node 3 (with managedOSVersionName method) to specified Teal version
```

Comment:
Maybe we can remove these commented lines?

Reply:
The idea is to re-add this test shortly, but yes I can remove them, we still have the history.

Reply:
done!